components/NavigationDrawer/src/MDCBottomDrawerPresentationController.h

```diff
@@ -87,4 +87,9 @@
  */
 @property(nonatomic, strong, nullable) UIColor *topHandleColor;
 
+/**
+ Whether the content/drawer reaches the full screen of the device.
```

Comment:
Is this `@YES` when the content *can* reach the window's height or when the drawer is currently *presented* to the height of the window?

If it's for the drawer's presentation, then I would suggest something like, "A boolean value that indicates whether the drawer is currently the full height of the window."

Either way, it's probably best to avoid mentioning "screen" and instead focus on the window height, since component's generally should not worry about screen dimensions.